README.md

```diff
@@ -1,3 +1,10 @@
+ᕙ(⇀‸↼‶)ᕗ
+
+Trello:
+  https://trello.com/b/DjEkOANi/michelle-ok-lambdamud
```

Comment:
Good job adding your trello links. It's a small thing but a lot of students forget to do it.